README.md

```diff
@@ -12,67 +16,92 @@ of your source files for more accurate results.
 
 ![](https://raw.githubusercontent.com/realm/SwiftLint/main/assets/screenshot.png)
 
-This project adheres to the [Contributor Covenant Code of Conduct](https://realm.io/conduct).
+This project adheres to the
+[Contributor Covenant Code of Conduct](https://realm.io/conduct).
 By participating, you are expected to uphold this code. Please report
 unacceptable behavior to [info@realm.io](mailto:info@realm.io).
 
-> Language Switch: [中文](https://github.com/realm/SwiftLint/blob/main/README_CN.md), [한국어](https://github.com/realm/SwiftLint/blob/main/README_KR.md).
+> Switch Language:
+> [中文](https://github.com/realm/SwiftLint/blob/main/README_CN.md)
+> [한국어](https://github.com/realm/SwiftLint/blob/main/README_KR.md)
+
+## Video Introduction
+
+To get a high-level overview of SwiftLint, we encourage you to watch this
+presentation recorded January 9th, 2017 by JP Simard (transcript provided):
+
+[![Presentation](https://raw.githubusercontent.com/realm/SwiftLint/main/assets/presentation.svg)](https://youtu.be/9Z1nTMTejqU)
 
 ## Installation
 
-### Using [Swift Package Manager](https://github.com/apple/swift-package-manager):
+### [Swift Package Manager](https://github.com/apple/swift-package-manager)
 
-> Replace `<version>` with the desired minimum version.
+SwiftLint can be used as a [command plugin](#swift-package-command-plugin)
+or a [build tool plugin](#swift-package-build-tool-plugins).
+
+Add
 
 ```swift
 .package(url: "https://github.com/realm/SwiftLint.git", from: "<version>")
 ```
 
-SwiftLint can be used as a [command plugin](#swift-package-command-plugin) or a [build tool plugin](#swift-package-build-tool-plugins).
+to your `Package.swift` file to consume the latest release of SwiftLint
+automatically or pin the dependency to a specific version:
+
+```swift
+.package(url: "https://github.com/realm/SwiftLint.git", exact: "<version>")
+```
+
+> [!IMPORTANT]
```

Comment:
I believe that can be a note instead. (allows the other important callouts to stand out more)
```suggestion
> [!NOTE]
```